.github/workflows/pr-check.yml

```diff
@@ -25,7 +74,39 @@ jobs:
 
       - run: yarn install --prefer-offline --frozen-lockfile
 
+  Build_Backend:
```

Comment:
Can we please just have 2 jobs: a `Backend` and `Frontend` job. Both of which should have a build and test step

Reply:
3 jobs - 
`Setup`
`Backend`
`Frontend`

Reply:
and `Analyse`

Reply:
> 3 jobs - `Setup` - install `Backend` - build and test `Frontend` - build and test

The latest one should be fine